README.md

```diff
@@ -1,2 +1,3 @@
 # vim-bats
-Run Bats tests from Vim.
+
+Run Bats tests from Vim. See the documentation by running `:help vim-bats`.
```

Comment:
The correct way to find the plugin documentation is `:help bats`, not `:help vim-bats`.